rllib/agents/pg/config_schema.py

```diff
@@ -0,0 +1,8 @@
+# Experimental.
```

Comment:
A jsonschema dict to check for type/bounds/etc. violations in our configs.
This will not stay here for PG (because PG does not add any own keys to COMMON_CONFIG), it's just here for discussion. For other algos, though, it'll look like this.

Reply:
IMO schema is not needed for config-- the few cases it helps are catching things like negative workers, which isn't something that users are confused by typically. On the flip side, it adds a lot of boilerplate that distracts from the actual config.

Reply:
Yeah, but that's why I separated the schema from the actual config (which is unchanged and still in pg.py). The user won't even see this unless she checks out this extra schema file.

Reply:
Please remove this for now, as we haven't decided whether to use schema. It's confusing to have this file exist, even if separate.

Reply:
done